static/js/main.js

```diff
@@ -32,7 +32,7 @@ var elements = $('.modal-overlay, .modal');
 $('.close-modal').click(function() {
     elements.removeClass('active');
 });
-var csrftoken = $('meta[name=csrf-token]').attr('content')
+var csrftoken = "lol"
```

Comment:
what's this for?

Reply:
To remove the csrf token to make deployment work